Add unit tests for common.GroupKey and utils.MD5

EricHetti opened the pull request:
I've analysed your codebase and noticed that `com.alibaba.nacos.client.config.common.GroupKey` and `com.alibaba.nacos.client.config.utils.MD5` is not fully tested.
I've written some tests for the methods in this class with the help of [Diffblue Cover](https://www.diffblue.com/opensource).

Hopefully, these tests will help you detect any regressions caused by future code changes. If you would find it useful to have additional tests written for this repository, I would be more than happy to look at other classes that you consider important in a subsequent PR.

petersear commented on client/src/test/java/com/alibaba/nacos/client/config/common/GroupKeyTest.java:
```diff
@@ -0,0 +1,57 @@
+    }
+
+    @Test
+    public void testParseKeyIllegalArgumentException() {
```
Change to three test methods for the three separate exceptions, only the first of which will be tested in the existing test method.

Braavos6 replied:
Done

Braavos6 replied:
Done

EricHetti committed:
Add unit tests for common.GroupKey and utils.MD5

Fully-qualified classname
com.alibaba.nacos.client.config.common.GroupKey
com.alibaba.nacos.client.config.utils.MD5

These tests were written using Diffblue Cover.
